.github/workflows/e2e-tests.yml

```diff
@@ -12,9 +12,14 @@ on:
 
 jobs:
   e2e:
-    name: " " # GitHub UI is noisy when calling reusable workflows, so use whitespace for name to reduce noise
+    name: E2E Tests - Shard ${{ matrix.shard }} of ${{ matrix.total_shards }}
```

Comment:
I don't believe ` ${{ matrix.shard }}` works here as you don't have access to the matrix var in the name. However you don't really need it as you can see which matrixed task is running:

![image](https://github.com/user-attachments/assets/20d765ae-ce47-4e5d-8c7a-e3fe373cadd3)

Reply:
I'm not sure why it was doing the above before - but it seems to be using the `e2e` job name now:
![image](https://github.com/user-attachments/assets/81ac8354-0c60-407d-b942-1e359b66b2fa)

https://github.com/navapbc/platform-test-nextjs/actions/runs/11695557464/job/32571326346